.php-cs-fixer.php

```diff
@@ -39,7 +39,6 @@
     ],
     'single_quote'                              => true,
     'ternary_operator_spaces'                   => true,
-    'trailing_comma_in_multiline'               => ['elements' => ['arrays']],
```

Comment:
why you say non-existent config value? https://cs.symfony.com/doc/rules/control_structure/trailing_comma_in_multiline.html